foundation-extensions/identity-for-edge-network/api-reference.md

```diff
@@ -359,9 +359,282 @@ The Identity for Edge Network extension does not read the Mobile SDK's privacy s
 
 See [`MobileCore.resetIdentities`](../mobile-core/mobile-core-api-reference.md#resetidentities) for more details.
 
+## setAdvertisingIdentifier
+When this API is called with a valid advertising identifier, the Identity for Edge Network extension includes the advertising identifier in the XDM Identity Map using the namespace `GAID` (Google Advertising ID) in Android and `IDFA` (Identifier for Advertisers) in iOS. If the API is called with the empty string (`""`), `null`/`nil`, or the all-zeros [UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier) string values, the advertising identifier is removed from the XDM Identity Map (if previously set).
+
+The advertising identifier is preserved between app upgrades, is saved and restored during the standard application backup process, and is removed at uninstall.
+
+{% tabs %}
+{% tab title="Android" %}
+
+{% hint style="warning" %} In order to enable collection of the user's current advertising tracking authorization selection for the provided advertising identifier, you need to install and register the [AEPEdgeConsent](https://aep-sdks.gitbook.io/docs/foundation-extensions/consent-for-edge-network) extension and update the [AEPEdge](https://aep-sdks.gitbook.io/docs/foundation-extensions/experience-platform-extension) dependency to minimum 1.3.2. {% endhint %}
```

Comment:
Wait, hint blocks work on a one liner?

Just for clarity, though (and for future easier migration purposes), can we split the {hint style} and the {endhint} blocks onto separate lines?

Reply:
Additionally, can we use a relative link for the links to the AEPEdgeConsent/AEPEdge API guides?